src/games/strategy/net/MacFinder.java

```diff
@@ -36,41 +41,26 @@ private static String GetMacAddress() {
       final InetAddress address = InetAddress.getLocalHost();
       final NetworkInterface localHostNI = NetworkInterface.getByInetAddress(address);
       if (localHostNI != null) {
-        // final byte[] rawMac = localHostNI.getHardwareAddress();
-        final Method m = NetworkInterface.class.getMethod("getHardwareAddress");
```

Comment:
@DanVanAtta also the code I added was commented out before... Probably, because this was intented when Triplea requires java 6+